ci/build-binaries-and-appimage.sh

```diff
@@ -40,11 +40,20 @@ OLD_CWD="$(readlink -f .)"
 pushd "$BUILD_DIR"
 
 # configure build and generate build files
-cmake "$REPO_ROOT" \
-    -DCMAKE_INSTALL_PREFIX=/usr \
+
+CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=/usr \
     -DCMAKE_BUILD_TYPE=RelWithDebInfo \
     -DBUILD_TESTING=ON \
-    -DAPPIMAGEKIT_PACKAGE_DEBS=ON
+    -DAPPIMAGEKIT_PACKAGE_DEBS=ON"
+
+if [[ $ARCH =~ loongarch ]]
+then
+    CMAKE_ARGS="$CMAKE_ARGS \
+        -DBUILD_TESTING=ON \
+        -DUSE_SYSTEM_LIBARCHIVE=ON"
+fi
+
+cmake "$REPO_ROOT" "$CMAKE_ARGS"
```

Comment:
The second variable is not expanded correctly. Fix `CMAKE_ARGS` to be a proper bash array. Then, use `${CMAKE_ARGS[@]}` to expand it in the call.

Reply:
done.